### DISCUSSION TITLE
Disable sensor/sensor interface

### DISCUSSION THREAD

**Discussion by wtl06639** [Security-Onion-Solutions/securityonion#3046] [Unsupported Versions] [answered]
i have a distributed setup with a manager and heavy nodes and would like to know how to be able to disable a specific sensor or sensor interface through the manager.  I wasn't able to find what i was looking for in documentation.

Thanks!

**Comment by TOoSmOotH** [comment, marked as answer]
If you want to disable a specific interface from being monitored remove it from the bond. If you want to remove a sensor you can reject it's salt key and reboot it. It will then be unable to start its services.
